plugins/parsers/csv/parser.go

```diff
@@ -169,6 +171,12 @@ func parseCSV(p *Parser, r io.Reader) ([]telegraf.Metric, error) {
 	for _, record := range table {
 		m, err := p.parseRecord(record)
 		if err != nil {
+			if p.SkipErrors {
+				if p.Log != nil {
+					p.Log.Infof("[parsers.csv] Malformed csv line: %s", err.Error())
```

Comment:
The logger should always be set-up correctly, if it isn't, then this is a bug. So there is no need to check for the logger to be nil and there is no need to prefix the error message as this is done by the logger
```suggestion
				p.Log.Infof("Malformed line: %v", err)
```
You might also think of making this a `warn`, but on the otherhand this also has the potential to spam the log file. Not exactly sure what to do here... Maybe only log the first malformed line per run?

Reply:
Good point, I agree Debug would be better than Warning or Error.

```suggestion
				p.Log.Debugf("parsing error: %v", err)
```

Reply:
Sorry for the delay... I was traveling for the holidays season. I also agree with the Debug statement. In my case, if I really need to check missing data, then I would enable the debug flag on the agent. I have updated with all the changes suggested. Thanks for the quick replies 👍 

Here is how it looks the messages when debug enabled:
`2021-12-22T11:36:30Z D! [parsers.csv::file] Parsing error: parsing time "corrupted_line" as "02/01/06 03:04:05 PM": cannot parse "corrupted_line" as "02"`